src/main/java/dao/impl/OrderMySQLDaoImpl.java

```diff
@@ -0,0 +1,73 @@
+package dao.impl;
+
+import dao.OrderDao;
+import model.Code;
+import model.Order;
+import model.User;
+import org.apache.log4j.Logger;
+import utils.DbConnector;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+public class OrderMySQLDaoImpl implements OrderDao {
+
+    private static final Logger logger = Logger.getLogger(OrderMySQLDaoImpl.class);
+
+    @Override
+    public Optional<Order> getOrder(User user) {
+        try (Connection connection = DbConnector.connect()) {
+            Statement statement = connection.createStatement();
+            String sql = String.format("SELECT `order`.id, id_user, id_code, id_basket, name, surname, city, adress, phone, email, password, role, code" +
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit
and better move queries in the class